charts/csi-baremetal-deployment/values.yaml

```diff
@@ -0,0 +1,159 @@
+# Docker registry to pull images
+global:
```

Comment:
Imo, we don't need most of these values, because they should be configured in CSI Deployment CR. I think, we should leave values, related to ConfigMaps, SCs and RBACs.

Reply:
@AnastasiaShemyakinskaya What values do you mean?
In my view, CSI Deployment CR is created with this chart and we should pass all values from chart into it

Reply:
Ok, in my view this CR should be created by user, but I think you are right.